### PR TITLE
Added new release alert

### LINKED ISSUE #469: Change log tracking and update publishing
**Describe the solution you'd like**
Keeping track of added features in a format which is understandable to our user base. Giving insight to what additions are made beyond the Github history. Preferably published to a location that is easy to access on our site, e.g. our landing page.

### PR DESCRIPTION
**Pull request by engelhamer**
## Description
Added an alert beneath the breadcrumb to show that a new version of eJournal has become available. This will be shown until a week after the release. If the user dismisses the alert they will not see it again until a new version is released (storing the version they don't want to see anymore in the database).

image

## Summary of changes
- Fixes #469

### CODE REVIEW

File: src/vue/src/views/Home.vue [hunk 1]
```diff
@@ -65,13 +65,13 @@ export default {
         }
     },
     components: {
-        'content-columns': contentColumns,
```
Comment (mjvkeulen) [inline src/vue/src/views/Home.vue]: :(

File: src/django/VLE/migrations/0020_preferences_hide_version_alert.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    dependencies = [
+        ('VLE', '0019_unlock_due_lock'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preferences',
+            name='hide_version_alert',
+            field=models.TextField(default='v0.0', max_length=10),
```
Comment (mjvkeulen) [inline src/django/VLE/migrations/0020_preferences_hide_version_alert.py]: Is this done for comparison reasons? Making it nullable would be might accurate, don't mind either way however.

Comment (engelhamer) [reply src/django/VLE/migrations/0020_preferences_hide_version_alert.py]: This does not carry any meaning. Ir can be changed to null indeed.

File: src/vue/src/components/assets/VersionAlert.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        return {
+            version: CurrentRelease.version,
+            message: CurrentRelease.message,
+            date: CurrentRelease.date
+        }
+    },
+    computed: {
+        showVersionAlert () {
+            return this.message !== '' &&
+                this.$store.getters['preferences/hideVersionAlert'] !== this.version &&
```
Comment (mjvkeulen) [inline src/vue/src/components/assets/VersionAlert.vue]: I prefer reading the syntax made available by mapGetters, but there still only a couple vuex uses in this component.

File: src/vue/src/components/assets/BreadCrumb.vue [hunk 4]
```diff
@@ -36,17 +36,20 @@
                 </span>
             </h1>
         </div>
+        <version-alert/>
```
Comment (Larspolo) [inline src/vue/src/components/assets/BreadCrumb.vue]: I would prefer this being added above the rest as its not part of the actual content

**Pull request merged**